### ISSUE TITLE
Issue: Blockmasters is messing with there API

### ISSUE THREAD

**Issue by Byrd1083** [RainbowMiner/RainbowMiner#151]
I got alerted this am that Blockmasters launched a new mobile app and there site is undergoing some changes. I noticed the rainbow api has stopped working concurring with there changes. Should take a look at it because rainbow is no longer mining from that pool.

![image](https://user-images.githubusercontent.com/42227809/45505055-6c71fe00-b740-11e8-9280-f7bd2461fda3.png)

**Comment by RainbowMiner** [comment]
Yes, they had a major outage. I will keep track of the changes (and let us hope, they finally fix their currencies API).
